Reformatting 'pyproject.toml'
--- pyproject.toml	(original)
+++ pyproject.toml	(reformatted)
@@ -6,18 +6,14 @@
 name = "whey"
 version = "2021.0.0"
 description = "A simple Python wheel builder for simple projects."
-keywords = [ "pep517", "pep621", "build", "sdist", "wheel", "packaging", "distribution",]
+keywords = [ "build", "distribution", "packaging", "pep517", "pep621", "sdist", "wheel",]
+dependencies = [ 'django>2.1; os_name != "nt"', 'django>2.0; os_name == "nt"', "gidgethub[httpx]>4.0.0", "httpx",]
 dynamic = [ "classifiers", "requires-python",]
-dependencies = [
-  "httpx",
-  "gidgethub[httpx]>4.0.0",
-  "django>2.1; os_name != 'nt'",
-  "django>2.0; os_name == 'nt'"
-]

 [[project.authors]]
+name = "Dominic Davis-Foster"
 email = "dominic@davis-foster.co.uk"
-name = "Dominic Davis-Foster"
+

 [project.urls]
 Homepage = "https://whey.readthedocs.io/en/latest"
