Reformatting 'pyproject.toml'
--- pyproject.toml	(original)
+++ pyproject.toml	(reformatted)
@@ -1,22 +1,18 @@
-[project]
-keywords = [ "pep517", "pep621", "build", "sdist", "wheel", "packaging", "distribution",]
-version = "2021.0.0"
-description = "A simple Python wheel builder for simple projects."
-dynamic = [ "classifiers", "requires-python",]
-dependencies = [
-  "httpx",
-  "gidgethub[httpx]>4.0.0",
-  "django>2.1; os_name != 'nt'",
-  "django>2.0; os_name == 'nt'"
-]
-name = "whey"
-
 [build-system]
 requires = [ "whey",]
 build-backend = "whey"

+[project]
+name = "whey"
+version = "2021.0.0"
+description = "A simple Python wheel builder for simple projects."
+keywords = [ "build", "distribution", "packaging", "pep517", "pep621", "sdist", "wheel",]
+dependencies = [ 'django>2.1; os_name != "nt"', 'django>2.0; os_name == "nt"', "gidgethub[httpx]>4.0.0", "httpx",]
+dynamic = [ "classifiers", "requires-python",]

-
+[[project.authors]]
+name = "Dominic Davis-Foster"
+email = "dominic@davis-foster.co.uk"


 [project.urls]
@@ -24,6 +20,7 @@
 Documentation = "https://whey.readthedocs.io/en/latest"
 "Issue Tracker" = "https://github.com/repo-helper/whey/issues"
 "Source Code" = "https://github.com/repo-helper/whey"
+
 [tool.whey]
 base-classifiers = [ "Development Status :: 4 - Beta",]
 python-versions = [ "3.6", "3.7", "3.8", "3.9", "3.10",]
@@ -31,7 +28,3 @@
 platforms = [ "Windows", "macOS", "Linux",]
 license-key = "MIT"

-[[project.authors]]
-email = "dominic@davis-foster.co.uk"
-name = "Dominic Davis-Foster"
-
